src/formatters/index.ts

```diff
@@ -3,23 +3,38 @@ import type { LoggerOptions } from 'pino';
 
 export const DEFAULT_MAX_OBJECT_DEPTH = 4;
 
+const DEFAULT_FUNCTIONS = true;
+const DEFAULT_STRING_LENGTH = 512;
+
 export interface FormatterOptions {
   /**
    * Maximum property depth of objects being logged. Default: 4
    */
   maxObjectDepth?: number;
 
+  /**
+   * Allows functions in log objects. Default: true
+   */
+  functions?: boolean;
```

Comment:
Maybe something like this?
```suggestion 
 /**
   * Controls how function properties are handled in log objects.
   * 
   * When `true`: Function properties are omitted from log output entirely.
   * 
   * When `false`: Function properties are represented as `"[Function]"` in logs. (default)
   * 
   * @example
   * ```typescript
   * // omitFunctions: false
   * logger.info({ fn: () => {}, data: 'value' });
   * // Output: { "fn": "[Function]", "data": "value" }
   * 
   * // omitFunctions: true
   * logger.info({ fn: () => {}, data: 'value' });
   * // Output: { "data": "value" }
   * ```
   */
  omitFunctions?: boolean;
```